config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
+  get '/posts', to: 'posts#index'
+  get '/posts/:id', to: 'posts#show'
+  post '/posts/create', to: 'posts#create'
```

Comment:
Post looks a lot like a resource. These 3 lines could be replaced with:
`resources :post`